tchannel-core/src/main/java/com/uber/tchannel/schemes/JSONSerializer.java

```diff
@@ -42,14 +42,12 @@
     @Override
     public String decodeEndpoint(ByteBuf arg1) {
         String endpoint = arg1.toString(CharsetUtil.UTF_8);
-        arg1.release();
```

Comment:
oh, it is released when request is done using.

https://github.com/uber/tchannel-java/pull/99/files#diff-0c30656be54ed88283a28294e0fa1555R183